frontend/package.json

```diff
@@ -96,6 +98,7 @@
     "react-table": "7.8.0",
     "react-test-renderer": "17.0.2",
     "react-timeago": "7.1.0",
+    "react-virtuoso": "^4.2.1",
```

Comment:
Only dependency we're adding is [react-virtuoso](https://github.com/petyosi/react-virtuoso)

Reply:
I just found https://github.com/bvaughn/react-virtualized-select which is something else we can look into